database/seeders/FacilitySeeder.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class FacilitySeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        DB::table('facilities')->insert([
```

Comment:
Idk which is the best case, but i think you can make a factory as a blue print so that when you make data it will be easier you can refer from my branch wdyt?

Reply:
previously i used factory for facilities but it's different like venue. i think for facilities need more context so, i just created seeder